File: frontend/src/features/actividades/revisar_inscripcion_cbr/models/revisar_inscripcion_cbr.ts
export interface RevisarInscripcionCbr {
  id_revisar_inscripcion_cbr: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  is_enviar_reparo: boolean;
  observaciones?: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/revisar_inscripcion_cbr.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class revisar_inscripcion_cbr : base_auditoria
    {
        public int id_revisar_inscripcion_cbr { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public bool is_enviar_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}
